### DISCUSSION TITLE
Use custom migrations directory for admin UI (Pocketbase as a framework)

### DISCUSSION THREAD

**Discussion by kisamoto** [pocketbase/pocketbase#2676] [Q&A] [answered]
When using Pocketbase as a framework, can register `migratecmd` and set a custom directory which works for creating migrations with the CLI (`go run main.go migrate create {{name}}` - *see below code for setting the directory*) however it seems this is not the same directory for migrations created with the Admin UI. When adding or modifying collections in the Admin UI it defaults to the top level `migrations/` directory.

Is there a configuration option to change this to use the same directory as the migrate command? 

Thanks in advance and for developing such an awesome tool.

```go
	migratecmd.MustRegister(app, app.RootCmd, &migratecmd.Options{
		Dir:         "./db/migrations", // path to migration files
		Automigrate: true,              // auto creates migration files when making collection changes
	})
```

**Comment by ganigeorgiev** [comment, marked as answer]
It is working fine for me locally with v0.16.4. Could you provide a more complete reproducible code sample?

In both cases the migrations, created via cli and with automigrate when mofifying collections from the Admin UI, are created in the specified directory.

**Reply by kisamoto** [reply]
Well that's embarrassing - I can't replicate in the trial project I wanted to send so I'll debug my own code. Thanks for the swift reply!
